<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClusterSettings</class>
 <widget class="QWidget" name="ClusterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>292</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding:1;margin:0</string>
  </property>
  <layout class="QGridLayout" name="centralGrid" columnstretch="0,0,0,0,0,0,0">
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="deleteBtn">
     <property name="toolTip">
      <string>Delete this cluster</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="clusterType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Cluster distribution type</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Uniform</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QComboBox" name="colorSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>42</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>28</width>
       <height>12</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Points' color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="showSelection">
     <property name="toolTip">
      <string>Show cluster</string>
     </property>
     <property name="text">
      <string>Adjust</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>P.#</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="pointCount"/>
     </item>
    </layout>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QPushButton" name="refillBtn">
     <property name="toolTip">
      <string>Regenerate points</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QWidget" name="normalDataArea" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>S.dev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="stdDev">
        <property name="styleSheet">
         <string notr="true">padding:1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Clip R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="clipRadius"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
